### ISSUE TITLE
Support STATEMENT_BLOCK as mixin call parameter

### ISSUE THREAD

**Issue by AlexPopov87** [racodond/sonar-css-plugin#54]
Hi,

I have an issue with parsing detached rulesets on 4.7 version.

E.g. 
```
.desktop-and-old-ie(@rules) {
  @media screen and (min-width: 1200px) { @rules(); }
  html.lt-ie9 &                         { @rules(); }
}

header {
  background-color: blue;

  .desktop-and-old-ie({
    background-color: red;
  });
}
```

http://lesscss.org/3.x/features/#detached-rulesets-feature

This will throw parse error.

Thanks

**Comment by racodond** [comment]
Hi @AlexPopov87,

Indeed, new features coming with version 3 are not supported for now.
I haven't studied yet the workload to support version 3 either.

If you're eager to contribute, I'd be glad to help you.

David

**Comment by racodond** [comment]
Hi @AlexPopov87,

Thanks for your feedback!
Could you please give a try to the fix: https://github.com/racodond/sonar-css-plugin/releases/tag/%2354 and tell me if it properly fixes all your issues?

Thank you!

David

**Comment by AlexPopov87** [comment]
Hi @racodond,

I can confirm that issue is fixed in 4.9.

Thanks you!

**Comment by racodond** [comment]
Hi @AlexPopov87,

Thanks for your feedback!
I just released version 4.9 that includes the fix of your issue: https://github.com/racodond/sonar-css-plugin/releases/tag/4.9

David
